common/common.go

```diff
@@ -486,3 +487,21 @@ func GetExitStatus(err error) (int, error) {
 	}
 	return -1, err
 }
+
+// GetArch returns the current architecture. It returns runtime.GOARCH if not on a arm device
+// and armv6l, armv7l, arm64 otherwise
+func GetArch() string {
+	arch := runtime.GOARCH
+	if arch == "arm" {
+		utsName := &syscall.Utsname{}
+		syscall.Uname(utsName)
+		arch = ""
+		for _, b := range utsName.Machine {
+			if b == 0 {
+				break
+			}
+			arch += string(b)
+		}
+	}
+	return arch
```

Comment:
I think it makes more sense to use [ToAppcOSArch](https://github.com/appc/spec/blob/master/schema/types/labels.go#L149) directly here.